accepted/PSR-6-cache-meta.md

```diff
@@ -300,7 +300,7 @@ The following sample code is recommended in order to enforce the type check on
 the expiresAt() method:
 
 ```php
-if (! ($expiration instanceof \DateTime || $expiration instanceof \DateTimeImmutable)) {
+if (! (is_null($expiration) || $expiration instanceof \DateTime || $expiration instanceof \DateTimeInterface)) {
```

Comment:
It is not a PSR-2 thing no. PSR-2 doesn't touch on operators and PSR-12 excludes ruling on the unary not operator. https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#6-operators